docs/_build/testing/test-tools.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Integration testing and validation tools
+order: 3
+layout: post-toc
+---
+
+# Testing Tools
+
+The Zapier platform provides a set of tools to help inform and validate your integration before pushing changes out to users.
+
+## Canary Testing
+
+Canary testing is a way to test new changes temporarily with real users in production with the goal of validating changes to ship new changes with more confidence and reducing bugs.
+These users are not informed or aware of the changes, as this is usually done at random and in small subsets to obtain a sample. Builders should have careful monitoring in place to watch for errors and rollback when necessary.
+
+### Prerequisites
+
+- Completed build of your Zapier integration, built from the CLI (as of Sep 23, 2024, this is a CLI only feature)
+- If you haven’t used Zapier before, you'll want to learn the basics in our [Zapier Getting Started Guide](https://zapier.com/learn/zapier-quick-start-guide/).
+
+You may want to use the canary tool when adding a new feature, or fixing a bug. For example, if you are planning to roll out a bug fix, you may want to test this out to see if the fix will work. The usual validation steps may include unit tests, and _[setting up Zaps for validation](https://platform.zapier.com/build/test-monitoring). 
+Unit tests will ensure your integration code functionality matches what you intended to do. Setting up Zaps will ensure the trigger or action works with the inputs you provide. The canary tool ensures your changes work for many existing live Zaps, with different inputs and outputs. 
+
+`zapier canary` provides a new way to validate your integration and build confidence that the change can work for all different types of Zap set ups.
+`zapier canary` allows you to set the version you want to test with, the version you want to replace with, the percentage of traffic, and a duration before the versions are rolled back.
+You can choose to delete the canary test before the duration is expired in case something catastrophic occurs.
```

Comment:
nit: maybe instead of catastrophic, we could say "unexpected"? Catastrophic to me kind of implies irreversible and v bad, where in this case, a failure isn't great but also isn't the end of the world